modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -46,6 +47,23 @@ For information on the standard (non-Group-based) distribution of replica vBucke
 The distribution of vBuckets across groups is exemplified below.
 In each illustration, all servers are assumed to be running the Data Service.
 
+[IMPORTANT]
```

Comment:
This whole section 'Important' is  super complicated. I suggest we simplify it to something as follows. Also place it after the discussion on equal and unequal groups (not right at the top of the section):

If you have fewer server groups than the total number of data copies (= 1 + number of replicas), only as many copies will be placed as the number of server groups. Essentially you may not get as many replicas as you set if you have fewer server groups. 
E.g. if you have 2 server groups, and set replicas = 2, you will only have one replica for each vbucket. But if you have 3 server groups you will have 2 replicas for each vbucket.
This behavior is a change from pre-7.1 behavior where we did not reduce the number of data copies to match the number of server groups.